### ISSUE TITLE
Training results

### ISSUE THREAD

**Issue by Porphy** [tianyu0207/PEBAL#34]
@tianyu0207 Your work is excellent. When I tried to reproduce the results using the cityscapes_best.pth pre-trained weights that you provided for training, I found that very good performance was already achieved on the validation set in the first epoch. Is this normal?
![image](https://github.com/tianyu0207/PEBAL/assets/92923476/f41a0fe2-4fe4-4107-84a7-b4e80ce2adb5)
I also tried other semantic segmentation models, such as DDRNet. However, when using it for training, the initial results were very poor, and it took a complete 40 epochs or more of training to improve. On the other hand, the initial results of the DeepLabv3+ model you used were already very good, and there wasn't a significant improvement after training compared to the first epoch. Is this reasonable?
![image](https://github.com/tianyu0207/PEBAL/assets/92923476/2415ed4d-5dd1-41fa-8a5d-52c6e0fa8d0b)

**Comment by yyliu01** [comment]
Hi @Porphy 

Thanks for the interest of our work. Based on the uploaded figures, I observed improvements in both AuPRC and FPR during training. The slow improvements for our architecture is probably caused by the bottleneck of the higher performance.

In my experience, different architectures may require slightly varied learning rates or batch sizes to converge. The Out-of-Distribution (OoD) performance could be influenced by the inlier performance of the original model. In addition, the cityscapes_best.pth is from other repository and we don't provide any weight for the closed set segmentation architectures.

Cheers,
Yuyuan 

P.S., our newest work [RPL](https://github.com/yyliu01/RPL) archives more robust results and it will definitely provide higher performance for all the benchmarks.
